src/pages/Products/components/ProductCardButton.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { ShoppingBag } from '@mui/icons-material'
+import { Button, Grid, IconButton } from '@mui/material'
+
+import Localizator from '../../../common/components/Localizator'
+
+const ProductCardButton = () => {
+  return (
+    <Grid display="flex" flexDirection="row" justifyContent="space-between">
```

Comment:
wrong Grid component usage, please replace with Stack component

Grid helps to build columns using flexbox or CSS grid